.github/workflows/gene-essentiality.yml

```
@@ -0,0 +1,32 @@
+name: Check gene essentiality with Hart 2015
+
+on:
+  pull_request:
+    branches:
+      - "main"
```

It might be safer to start by allowing this workflow to run also in PRs targeting `develop`:
```suggestion
      - "main"
      - "develop"
```